manager/integration/tests/backupstore.py

```diff
@@ -17,14 +17,104 @@
 from common import is_backupTarget_nfs
 from common import get_longhorn_api_client
 from common import delete_backup_volume
+from common import get_backupstore_url
 
 BACKUPSTORE_BV_PREFIX = "/backupstore/volumes/"
 BACKUPSTORE_LOCK_DURATION = 150
 
-
 TEMP_FILE_PATH = "/tmp/temp_file"
 
 
+@pytest.fixture
+def backupstore_s3(client):
+    set_backupstore_s3(client)
+    yield
+    reset_backupstore_setting(client)
+
+
+@pytest.fixture
+def backupstore_nfs(client):
+    set_backupstore_nfs(client)
+    yield
+    reset_backupstore_setting(client)
+
+
+@pytest.fixture(params=["s3", "nfs"])
```

Comment:
Good stuff! I learnt a new thing from you today :D

Reply:
Thank you! :smile: